x-pack/plugins/security_solution/public/app/app.tsx

```diff
@@ -79,11 +81,22 @@ const StartAppComponent: FC<StartAppComponent> = ({ subPlugins, ...libs }) => {
   const { i18n } = useKibana().services;
   const history = createHashHistory();
   const libs$ = new BehaviorSubject(libs);
+  const storage = createSecuritySolutionStorage(localStorage);
 
   const store = createStore(
-    createInitialState(subPluginsStore.initialState),
+    createInitialState({
```

Comment:
maybe we could move that logic to the epic? In my opinion `app` shouldn't contain subplugin-specific logic

Reply:
I think this logic belong in this file `x-pack/plugins/security_solution/public/timelines/index.ts`

Reply:
and we can add a new param to return from there